ecommerce/README.rst

```diff
@@ -0,0 +1,27 @@
+edx Themes for Otto
+===================
+This directory contains themes for Otto, following directory structure should be followed for each theme
+
+
+.. code-block:: text
+
```

Comment:
@lamagnifica No, e-commerce theme can have the same name as an LMS/Studio theme. The reason is that e-commerce and LMS/Studio run in different environments and do not affect one another.